modules/auxiliary/scanner/ssh/ssh_login_pubkey.rb

```diff
@@ -175,7 +175,7 @@ def run_host(ip)
           create_credential_login(credential_data)
           tmp_key = result.credential.private
           ssh_key = SSHKey.new tmp_key
-          session_setup(result, scanner, ssh_key.fingerprint, credential_core.private_id) if datastore['CreateSession']
+          session_setup(result, scanner, ssh_key.fingerprint, result.credential.private) if datastore['CreateSession']
```

Comment:
Note to the tester, we'll need to verify that `result.credential.private` is a string rather than an active record object 🕵️ 
Will need to drop in a breakpoint and just double check this behavior before merging, potentially something akin to:
```
credential_core.private.id
```

Reply:
![image](https://user-images.githubusercontent.com/19910435/123443716-0fccee80-d5ce-11eb-8510-a65fc0f0f6bd.png)
Looks like a string to me